src/api/app/models/event/report_for_user.rb

```diff
@@ -1,10 +1,10 @@
 module Event
   class ReportForUser < Report
     self.description = 'Report for a user created'
-    payload_keys :user_login
+    payload_keys :accused
```

Comment:
What about `offender`?

Reply:
I'm not a native speaker but I guess you are only an offender if the accusations against you are found to be true...

Reply:
@srbaker ?

Reply:
> I'm not a native speaker but I guess you are only an offender if the accusations against you are found to be true...

Hm, didn't know that.